README.md

```diff
@@ -2,7 +2,7 @@
 
 Contributors: lukecarbis, ryankienstra, Stino11, rheinardkorf, studiopress, wpengine
 Tags: gutenberg, blocks, block editor, fields, template
-Requires at least: 5.0
+Requires at least: 5.4
```

Comment:
This might need more discussion. But it looks like 5.3 and earlier don't support hooks. 

So I don't think we'll be able to keep backwards-compat without giving up a lot of features.